src/notation/view/percussionpanel/percussionpanelmodel.cpp

```diff
@@ -263,6 +306,61 @@ void PercussionPanelModel::setUpConnections()
         case PanelMode::Mode::SOUND_PREVIEW: playPitch(pitch);
         }
     });
+
+    if (!audioSettings()) {
+        return;
+    }
+
+    audioSettings()->settingsChanged().onNotify(this, [this](){
```

Comment:
A slightly more effective way would be to add a separate channel to IProjectAudioSettings and connect to it:

async::Channel<engraving::InstrumentTrackId> trackInputParamsChanged() const

That way we won't have to react to every change in the mixer